src/ServiceControl.Management.PowerShell/Cmdlets/ServiceControlInstances/InvokeServiceControlInstanceUpgrade.cs

```diff
@@ -153,8 +156,8 @@ void PerformUpgrade(ServiceControlInstance instance, UnattendServiceControlInsta
             {
                 SkipQueueCreation = SkipQueueCreation,
                 DisableFullTextSearchOnBodies = DisableFullTextSearchOnBodies,
-                UpgradeInfo =
-                    UpgradeControl.GetUpgradeInfoForTargetVersion(installer.ZipInfo.Version, instance.Version),
+                UpgradeInfo = UpgradeControl.GetUpgradeInfoForTargetVersion(installer.ZipInfo.Version, instance.Version),
```

Comment:
Note that the continued references to `installer.ZipInfo.Version` are silly. Every installer and zip available in the app is going to be the same version, and that version is easily accessible using `Constants.CurrentVersion`. Using the `installer` for that just makes all this code harder to refactor.

Reply:
The code doesn't compile on this so I just removed this whole line